config/services.yaml

```
@@ -1306,6 +1306,9 @@ services:
     App\Event\UserStatusChangedEvent:
         autowire: false
 
+    App\Event\ReadStatusWillChangeEvent:
```

Maybe use Pre and Post for naming?

```suggestion
    App\Event\ReadStatusPreChangeEvent:
```